kustomize/README.md

```diff
@@ -8,7 +8,7 @@ Kustomize is a Kubernetes configuration management tool that allows users to cus
 
 ## Prerequisites
 
-You need to have a Kubernetes cluster where you will deploy the Online Boutique's Kubernetes manifests into, you can follow the instruction [here](/).
+You need to have a Kubernetes cluster where you will deploy the Online Boutique's Kubernetes manifests. To set up a GKE (Google Kubernetes Engine) cluster, you can follow the instruction in the [root `/README.md`](/).
```

Comment:
My rationale for change: Avoid "here" for links because it doesn't provide enough meaning about the link. [Source](https://youtu.be/sXj0r4z5B5M?t=773).

Reply:
Good catch/point.